easybuild/easyconfigs/p/PICRUSt2/PICRUSt2-2.5.2-foss-2022b.eb

```diff
@@ -32,8 +32,18 @@ exts_list = [
     }),
 ]
 
+postinstallcmds = [
+    'cp -r %(builddir)s/%(name)s/%(namelower)s-%(version)s/%(namelower)s/default_files ' +\
```

Comment:
another option here is to use the `start_dir` template:

```
cd %(start_dir)s && cp -a %(namelower)s/default_files ...
```

Reply:
using %(start_dir)s  will not significally shorten the line since  %(builddir)s/ == %(start_dir)s.

Reply:
Yeah, of course, I overlooked the extension aspect here...